pallets/asset/src/lib.rs

```diff
@@ -305,6 +305,13 @@ pub struct ClassicTickerRegistration {
     pub is_created: bool,
 }
 
+// Fiat Currency Struct
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
+pub struct FiatCurrency<String> {
+    pub codes: Vec<String>,
+}
```

Comment:
Let's move this definition into `fn currency_codes(...)` since it's only ever used there, so that you don't need to jump around when reading `fn currency_codes`.

Reply:
Alright would make the change